hack/deploy-clean.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -x
+
+HACK_DIR=${HACK_DIR:-$(dirname $0)}
+
+# Remove SBR finalizers
+$HACK_DIR/remove-sbr-finalizers.sh
+
+# Delete deployed resources
+kubectl delete -f deploy/crds/apps_v1alpha1_servicebindingrequest_cr.yaml
+kubectl delete -f deploy/crds/apps.openshift.io_servicebindingrequests_crd.yaml
+kubectl delete -f deploy/operator.yaml
+kubectl delete -f deploy/role_binding.yaml
+kubectl delete -f deploy/role.yaml
+kubectl delete -f deploy/service_account.yaml
```

Comment:
how about replacing this with:

```
kubectl delete -R -f deploy
```

Reply:
The `-R` would attempt to delete all of the yaml files under the `deploy` directory including those that are not Kubernetes resources, at all. Such as `deploy/olm-catalog/csv-config.yaml` and that would always be with error. We could of course move the `olm-catalog` specific directory somewhere else. But that would require additional changes at places that expect those files at that location, and those changes would be un-related to the point of this PR.

Reply:
alright